catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -52,7 +56,8 @@ const SearchForm = ({
   searchParams,
 }: Props) => {
   const { query, workType } = searchParams;
-
+  const searchForm = useRef();
+  const getForm = () => searchForm;
```

Comment:
Can you explain the need for this to me when you have a sec (i.e. why can't we just use the `searchForm ref`)?